pkg/server/endpoints/run.go

```diff
@@ -108,37 +105,8 @@ func (e *Endpoints) runUDSServer(ctx context.Context) error {
 	}
 }
 
-func (e *Endpoints) addHandlers(ctx context.Context) {
+func (e *EndpointHandler) addHandlers(ctx context.Context) {
+	e.createMemberHandler(ctx)
```

Comment:
nit: I'd suggest to separate handlers from endpoint definitions, something like this

```suggestion
	http.HandleFunc("/createMember", handleCreateMember)
	http.HandleFunc("/createRelationship", handleCreateRelationship)
	// etc
```